src/styles/utils/_keyframes.scss

```diff
@@ -0,0 +1,73 @@
+@keyframes movetop {
+  0% {
+    top: -420px;
+    left: 0;
```

Comment:
If the 'left' property is not changing throughout the animation, it is not necessary to declare it in the keyframes. This applies to all the animations in this file.